tests/fixtures/express-openapi3/server.ts

```diff
@@ -3,6 +3,7 @@ import * as express from 'express';
 import * as methodOverride from 'method-override';
 import '../controllers/rootController';
 
+import '../controllers/optionsController';
```

Comment:
Can you add integration tests for the new routes in tests/integration/*.spec.ts

Reply:
@WoH I'm not sure what kind of tests you want.  
I made OPTIONS decorator base on HEAD and DELETE method, but I didn't find any kind of tests inside each integration tests.  
Maybe you wish to have a more proper test cases for OPTIONS and move on to HEAD and DELETE and so on, can you give me any clues to write such test cases in integration tests? thx.

Reply:
You can add a spec below [this test](https://github.com/lukeautry/tsoa/blob/master/tests/integration/express-server.spec.ts#L217) that looks roughly like this:

```ts
  it('correctly handles OPTIONS Requests', () => {
    const path = basePath + '/OptionsTest/Current';
    return verifyRequest(
        (err, res) => {
          expect(res).to.equal('');
          expect(err).to.equal(undefined);
        },
        request => request.options(basePath),
        201,
      );
  });
```

to the express, koa and hapi integration specs.